src/css/map.css

```diff
@@ -38,3 +39,25 @@
   left: 12px;
   z-index: 1;
 }
+
+.popup-text-header {
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 200%;
+  color: #000000;
+}
+
+.popup-text {
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 250%;
+  color: #000000;
+}
+
+.popup-root {
+  padding: 15px;
```

Comment:
I'm guessing you copied 15px padding from the Figma design, but there's also padding on `.mapboxgl-popup-content` which ends up making this too much padding. It'd be better to set `padding: 15px;` on `.mapboxgl-popup-content` using a more specific selector after you give the popup its own className.